packages/cli/src/commands/config/get.ts

```diff
@@ -10,6 +10,7 @@ export default class Get extends Command {
 
   async run() {
     const cfg = await new Config(this.config.configDir, this.error).get();
+    delete cfg.accessToken;
```

Comment:
Delete the access token from the config before printing it to console with `coveo config:get`

Reply:
Does that mean we have to run `auth:login` after doing a `config:get` command?

Reply:
No, it just means we delete it from the object we use to display in the console.

It does not actually remove it from the config, on your disk.